frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -123,7 +184,7 @@ func ListTargets(ctx context.Context, dt []byte) (*targets.List, error) {
 	return l, nil
 }
 
-func toDispatchState(ctx context.Context, dt []byte, opt ConvertOpt) (*dispatchState, *binfotypes.BuildInfo, error) {
+func toDispatchState(ctx context.Context, dt []byte, opt ConvertOpt) (*dispatchState, *dispatchStates, *binfotypes.BuildInfo, error) {
```

Comment:
You don't need extra param if you just store the sbom info you need to `dispatchState` properties.